lib/src/base-styles/_nx-page.scss

```diff
@@ -25,6 +25,7 @@
     overflow: hidden;
     text-overflow: ellipsis;
     white-space: nowrap;
+    max-width: max-content;
```

Comment:
`unset` would probably be simpler

Reply:
> 

Thanks! I updated it here: 1ba2b3704241535dd70661b3e257907df46c96b6